src/server/graphql/rootQuery.js

```diff
@@ -1,25 +1,18 @@
 import {GraphQLObjectType} from 'graphql';
-import agenda from './models/AgendaItem/agendaItemQuery';
-import meeting from './models/Meeting/meetingQuery';
-// import {nodeField} from './models/Node/nodeQuery';
 import organization from './models/Organization/organizationQuery';
 import teamMember from './models/TeamMember/teamMemberQuery';
 import user from './models/User/userQuery';
-import {toGlobalId} from 'graphql-relay';
 import User from 'server/graphql/types/User';
 
 export default new GraphQLObjectType({
   name: 'Query',
   fields: () => ({
-    // node: nodeField,
     viewer: {
       type: User,
       resolve: (source, args, {authToken}) => ({
-        id: toGlobalId('User', authToken.sub)
+        id: authToken.sub
       })
     },
-    ...agenda,
-    ...meeting,
```

Comment:
yep! everything with a spread operator is the old format. as i switch stuff to relay, I remove the legacy stuff. The relay convention is to store this stuff on the "viewer" (read User). so it'd be like:
```
viewer {
  team(teamId: "123") {
    agendaItems {
      id
      content
    }
  }
}
```

it's actually pretty great, relations are a lot easier to see in code & since everything is stored under the viewer you never need to check for the userId authorization because that's baked in.